packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -634,6 +634,8 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
 
     protected abstract getInitialSnapshotDetails(): Promise<ISnapshotDetails>;
 
+    public abstract setRoot(): void;
```

Comment:
i don't like this api, as calling it for the wrong reason will have unintended side effects. is this part of the final design, or just something we need through staging? as in will we eventually be able to get rid of the root concept and just use the alias blob to figure this out?

regardless can we add a comment here about what it does and what is for and that generally it should not be called

Reply:
i don't want to block this, but please at least add the comment before commit

Reply:
I'll add a comment. Yes, this is a 'staging' type of API.